### PR TITLE
Hackathon landing page

### PR DESCRIPTION
**Pull request by easyrun42**

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/hackathon/solanathon**](https://thirdweb-qasxpscj0.thirdweb-preview.com/hackathon/solanathon) |
|:-:|
|<a href="https://thirdweb-qasxpscj0.thirdweb-preview.com/hackathon/solanathon">Screenshot of /hackathon/solanathon</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/zHxPfcV9gtXizuE_Y5SpJGECSAlm72H5V81M3CefAl-xO2JLjlFIaXX1PDpMSWtNcDsFFiVSpL6XfISRNlIdMw.png">(view full size)</a> |







Commit 81e6c6e51b8619df1a31a2d4a1744ab3861ad4ed (https://thirdweb-qasxpscj0.thirdweb-preview.com).

File: components/hackathon/HackathonFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        textAlign="center"
+      >
+        DECEMBER 8 - 10
+      </Heading>
+
+      <LinkButton
+        href="https://docs.google.com/forms/d/1CT8LPG1DrcpTKAW38ScVzTcyBNwS6ANOk6xpfjpXLAs/edit"
+        onClick={() =>
+          trackEvent({
+            category: "solanathon",
```
Comment (nachoiacovino) [inline components/hackathon/HackathonFooter.tsx:49]: this category is wrong I believe?

File: components/hackathon/Reason.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          and meet VCs & industry leaders from{" "}
+          <b>Coinbase, Haun Ventures, Founders Inc</b>. and&nbsp;
+          <b>Framework Ventures.</b>
+        </Text>
+      </Flex>
+    </Container>
+  );
+};
+
+export default Reason;
```
Comment (nachoiacovino) [inline components/hackathon/Reason.tsx:36]: let's never export default for components

File: components/hackathon/Sponsors.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        columns={{ base: 2, md: 4 }}
+        gap={8}
+        placeItems="center"
+      >
+        {sponsors.map((sponsor) => (
+          <TrackedLink
+            key={sponsor.name}
+            href={sponsor.link}
+            isExternal
+            category="solanathon"
```
Comment (nachoiacovino) [inline components/hackathon/Sponsors.tsx:45]: wrong category?

File: components/hackathon/Timer.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            {value < 10 ? `0${value}` : value}
+          </Text>
+          <Text color="white">{value === 1 ? label : `${label}s`}</Text>
+        </Flex>
+      ))}
+    </Flex>
+  );
+};
+
+export default Timer;
```
Comment (nachoiacovino) [inline components/hackathon/Timer.tsx:74]: no export default

File: page-id.ts [hunk 5]
```diff
@@ -103,8 +103,8 @@ export enum PageId {
   // thirdweb.com/chainlist
   ChainsLanding = "chains-landing",
 
-  // thirdweb.com/hackathon/solanathon
-  // SolanaHackathonLanding = "solanathon",
+  // thirdweb.com/hackathon
+  HackathonLanding = "hackathon",
```
Comment (nachoiacovino) [inline page-id.ts:107]: this looks wrong? should be something related to BaseConsumerCrypto not generic hackathon

File: pages/hackathon/base-consumer-crypto.tsx [hunk 6]
```diff
@@ -0,0 +1,296 @@
...
+  {
+    ssr: false,
+  },
+);
+
+const Timer = dynamic(() => import("components/hackathon/Timer"), {
+  ssr: false,
+});
+
+const TRACKING_CATEGORY = "hackathon";
```
Comment (nachoiacovino) [inline pages/hackathon/base-consumer-crypto.tsx:31]: same for this, should be more specific than "hackathon"

File: pages/hackathon/base-consumer-crypto.tsx [hunk 7]
```diff
@@ -0,0 +1,296 @@
...
+              <Heading size="title.2xl" textAlign="center">
+                Guidelines
+              </Heading>
+              <Flex flexDir="column" gap={4} maxW={907}>
+                <Text size="body.xl" color="white" fontWeight="bold">
+                  To be eligible to win the hackathon, submitted projects must
+                  fulfill the following requirements:
+                </Text>
+                <Text size="body.xl">
+                  <List color="white">
```
Comment (nachoiacovino) [inline pages/hackathon/base-consumer-crypto.tsx:207]: If you do UnorderedList the dots get added automatically

File: pages/hackathon/base-consumer-crypto.tsx [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+                  size="body.xl"
+                  color="white"
+                  fontWeight="bold"
+                  textAlign="left"
+                >
+                  Our judges will grade submissions across 3 equally-weighted
+                  categories:
+                </Text>
+                <Text size="body.xl" color="white">
+                  <List>
```
Comment (nachoiacovino) [inline pages/hackathon/base-consumer-crypto.tsx:264]: OrderedList for the numbers to get added automatically

File: pages/hackathon/base-consumer-crypto.tsx [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+                    <ListItem>
+                      <b>1. Usability:</b> How useful or valuable is the
```
Comment (nachoiacovino) [inline pages/hackathon/base-consumer-crypto.tsx:266]: use strong instead of b

**Review by nachoiacovino — COMMENTED** [review]
mostly nits! but I think the tracking part is important for future reference

**Pull request merged**